src/less/utility/base/utility.less

```diff
@@ -63,6 +63,21 @@
     }
 }
 
+//greys out space in the li not taken by the img, in order to match height to other img
+// .image-treatment {
+//     .img {
+//         display: inline-block;
+//         object-fit: contain;
+//         max-width: 100%;
+//     }
+
+//     .li {
+//         background-color: rgba(0, 0, 0, 0.05);
+//         display: flex;
+//         justify-content: center;
+//     }
+// }
+
```

Comment:
Should remove this commented code

Reply:
As a utility, it should be:
```
.image-treatment {
    background-color: rgba(0, 0, 0, 0.05);
    display: flex;
    justify-content: center;

    > img {
        display: inline-block;
        object-fit: contain;
        max-width: 100%;
    }
}
```

Reply:
We'll need a better class name though :-)